metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppet-fail2ban",
-  "version": "4.2.1-rc0",
+  "version": "5.0.0",
```

Comment:
please don't bump the version, we will do it in the release PR.

Reply:
Ah yes my fault. I was planing to do a release, and that started to drop OS. But forgotten this. thank you for your eyes.